data/sample_incidents.js

```diff
@@ -0,0 +1,39 @@
+const allIncidents = [
```

Comment:
Good job adding multiple evidence to the incidents.  This is a great improvement over our past sample data setup